lib/charms/layer/jenkins/api.py

```diff
@@ -136,7 +136,26 @@ def restart(self):
         self.wait()
         unitdata.kv().set("jenkins.last_restart", time.time())
 
+    def quiet_down(self):
+        """
```

Comment:
The format for multiline docstrings would be (along with some suggested wording changes):

```
"""Put Jenkins in Quiet mode, in preparation for restart.

In this mode Jenkins won't start any builds."""
```